source/localizable/feedspec/index.en.html.md.erb

```diff
@@ -558,6 +560,7 @@ location. If this is not the case , then this attribute should be blank.
 long as they are accurate every time. If, even for one product, shipping costs
 are not accurate then they will be removed from all products in your data feed.
 
+> Shipping costs are exclusively for redirection.
```

Comment:
minor: `Shipping costs are taken into consideration exclusively for the redirection billing model.`